src/object/custom_particle_system.cpp

```diff
@@ -1241,4 +1246,1024 @@ CustomParticleSystem::ease_value(float* value, float target, float time, easing
   script_easings.push_back(req);
 }
 
+void
+CustomParticleSystem::spawn_particles(int amount, bool instantly)
+{
+  if (instantly)
```

Comment:
I *kind of* would invert the order of the if statements (`return;` early after the warning, but that might be personal preference).